ada/ast.py

```diff
@@ -13962,6 +13962,34 @@ def family_type():
         """
         return Entity.spec.family_type.cast(TypeExpr)._.designated_type
 
+    @langkit_property(public=True)
```

Comment:
Personally I prefer specifying the return type for all properties, especially public ones (avoid changing the API by mistake).